### ISSUE TITLE
[FE] Conditionally disable write method depending on connected chain

### ISSUE THREAD

**Issue by bitbeckers** [hypercerts-org/hypercerts#1248]
### Describe the feature you'd like to request

As a user, I want the able to execute write methods on hypercerts I own, when connected to the correct chain, so that I can manage my collection in the app.
As a user, I want to be notified when the SDK client is connected to a different chain than the hypercerts want to mutate is on, so that I cannot send, split, or burn the wrong fractions

### Describe the solution you'd like

Modal to confirm state changes in FE

### Describe alternatives you've considered

.

**Comment by bitbeckers** [comment]
Dependent on #1247

**Comment by Jipperism** [comment]
@holkexyz Closing as won't do, as SDK supports it but plasmic frontend won't. Feel free to reopen if it needs to be done.

**Comment by bitbeckers** [comment]
Isn't this already in the SDK and the 'CheckWritable' wrapper component?
